### PR TITLE
Addition of ANC translations

### PR DESCRIPTION
**Pull request by SebaMutuku**

### CODE REVIEW

File: android-json-form-wizard/src/main/java/com/vijay/jsonwizard/utils/NativeFormLangUtils.java [hunk 1]
```diff
@@ -91,6 +91,18 @@ public static String getTranslatedString(String str, Context context) {
         return translateString(str, ResourceBundle.getBundle(translationsFileName, currLocale));
     }
 
+    public static String getTranslatedANCString(String key, Context context) {
+        //anc_profile.step4.allergies.options.albendazole.text
```
Comment (dubdabasoduba) [inline android-json-form-wizard/src/main/java/com/vijay/jsonwizard/utils/NativeFormLangUtils.java:95]: Remove dead code

File: android-json-form-wizard/src/main/java/com/vijay/jsonwizard/utils/Utils.java [hunk 2]
```diff
@@ -821,7 +822,20 @@ public static JSONObject getRepeatingGroupCountObj(@NotNull WidgetArgs widgetArg
...
+            JSONArray options = item.getJSONArray(JsonFormConstants.OPTIONS_FIELD_NAME);
+            JSONObject selectedOption = formUtils.getOptionFromOptionsUsingKey(options, value);
+            newValue.put(JsonFormConstants.VALUE, value);
+            newValue.put(JsonFormConstants.TEXT, selectedOption.optString(JsonFormConstants.TRANSLATION_TEXT, ""));
+            return newValue;
+        }
+        newValue.put(JsonFormConstants.VALUE, value);
+        newValue.put(JsonFormConstants.TEXT, item.optString(JsonFormConstants.TRANSLATION_TEXT, ""));
+        return newValue;
+    }
```
Comment (dubdabasoduba) [inline android-json-form-wizard/src/main/java/com/vijay/jsonwizard/utils/Utils.java:838]: Please optimize this method. Line `825` to `838` and `985` to `997` can be in the same method

**Review by dubdabasoduba — CHANGES_REQUESTED** [review]

File: android-json-form-wizard/src/main/java/com/vijay/jsonwizard/utils/Utils.java (hunk 2 above)
Comment (SebaMutuku) [reply android-json-form-wizard/src/main/java/com/vijay/jsonwizard/utils/Utils.java]: Done

File: android-json-form-wizard/src/main/java/com/vijay/jsonwizard/utils/NativeFormLangUtils.java (hunk 1 above)
Comment (SebaMutuku) [reply android-json-form-wizard/src/main/java/com/vijay/jsonwizard/utils/NativeFormLangUtils.java]: Done

**Review by dubdabasoduba — APPROVED** [review]

**Pull request merged**